src/client/api/event.c

```diff
@@ -623,11 +623,18 @@ daos_eq_create(daos_handle_t *eqh)
 		return -DER_NOMEM;
 
 	eqx = daos_eq2eqx(eq);
+
+	rc = crt_context_create(&eqx->eqx_ctx);
+	if (rc) {
+		D_WARN("Failed to create CART context ("DF_RC"), "
```

Comment:
DF_RC can be quite a lot of text, and the style guide says it should be at the end of the line, immediately prior to to the \n.

Reply:
OK i thought this was just the RC (string) error code printed. anyway, i just moved it to the end.